File: 240611/특정 위치의 문자/char-in-specific-location.py
input_chr = input()

chr_lst = ['L', 'E', 'B', 'R', 'O', 'S']
if input_chr in chr_lst:
    idx = 0
    for elem_idx, elem in enumerate(chr_lst):
        if elem == input_chr:
            idx = elem_idx
            break
    print(idx)
else:
    print("None")

File: 251117/특정 위치의 문자/char-in-specific-location.cpp
#include <iostream>
#include <string>
#include <vector>
using namespace std;

int main() 
{
    // Please write your code here.
    string input_chr;
    cin >> input_chr;

    vector<string> chr_lst = {"L", "E", "B", "R", "O", "S"};

    bool found = false;
    int idx = 0;

    for(int i = 0; i < chr_lst.size(); i++)
    {
        if(chr_lst[i] == input_chr)
        {
            idx = i;
            found = true;
            break;
        }
    }

    if(found)
        cout << idx << endl;
    else
        cout << "None" << endl;
    return 0;
}